test/common/quic/envoy_quic_h3_fuzz_helper.h

```diff
@@ -26,15 +26,29 @@ class H3Packetizer {
 // would be sent over the wire.
 class QuicPacketizer {
 public:
+  struct QuicPacket {
+    size_t size{0};
+    char payload[1460];
+  };
+
   QuicPacketizer(const quic::ParsedQuicVersion& quic_version,
                  quic::QuicConnectionHelperInterface* connection_helper);
-  void serializePackets(const test::common::quic::QuicH3FuzzCase& input);
-  void foreach (std::function<void(const char*, size_t)> cb);
+  size_t serializePackets(const test::common::quic::QuicH3FuzzCase& input, QuicPacket* packets,
```

Comment:
Can you make it return std::vector<QuicPacket> instead of using output argument?